enclave_manager/avalon_enclave_manager/kme/kme_enclave_info.py

```diff
@@ -22,31 +22,31 @@
 from requests.exceptions import Timeout
 from requests.exceptions import HTTPError
 import utility.hex_utils as hex_utils
+import utility.file_utils as file_utils
 import avalon_enclave_manager.kme.kme_enclave as enclave
 import avalon_enclave_manager.base_enclave_info as enclave_info
-from avalon_enclave_manager.base_enclave_info import BaseEnclaveInfo
 
 logger = logging.getLogger(__name__)
 
 
-class KeyManagementEnclaveInfo(BaseEnclaveInfo):
+class KeyManagementEnclaveInfo(enclave_info.BaseEnclaveInfo):
     """
     KME info class to initialize enclave, signup enclave and hold
     data obtained post signup.
     """
 
     # -------------------------------------------------------
-    def __init__(self, config):
+    def __init__(self, config, worker_id):
 
         # Initialize the keys that can be used later to
         # register the enclave
         self._config = config
+        self._worker_id = worker_id
         enclave._SetLogger(logger)
-        super(KeyManagementEnclaveInfo, self).__init__(
-            enclave.is_sgx_simulator())
+        super().__init__(enclave.is_sgx_simulator())
```

Comment:
Can we add a comment why the lines above super constructor has to be called before calling super constructor ? Does super class use these member variables

Reply:
Not really. Though I see a scope of better handling o config which I am willing to take up in another PR.